week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -8,8 +8,15 @@
   NOTE: don't include any element that is not a "family".
 */
 
-function colonisers() {
 
+
+function colonisersName(strrName) {
```

Comment:
A small point about the name of this function: normally we try to name our functions so that what the function returns makes sense when we call the function. Think carefully about what type this function returns. Do you think it makes sense to call it `colonisersName`? Or would another name make sense?